include/ruby/internal/abi.h

```diff
@@ -24,7 +24,7 @@
  * In released versions of Ruby, this number is not defined since teeny
  * versions of Ruby should guarantee ABI compatibility.
  */
-#define RUBY_ABI_VERSION 0
+#define RUBY_ABI_VERSION 1
```

Comment:
You don't have to bump up this, because the size of `rb_data_type_t` does not change.
Or, extend `reserved` for the future use, if you have any plan to add other functions?